seqr/models.py

```diff
@@ -1011,19 +1011,48 @@ def __unicode__(self):
     def _compute_guid(self):
         return 'VSR%07d_%s' % (self.id, _slugify(str(self)))
 
-class DeletableSampleMetadataModel(models.Model):
 
-    sample = models.ForeignKey('Sample', on_delete=models.CASCADE, db_index=True)
-    gene_id = models.CharField(max_length=20)  # ensembl ID
+class BulkOperationBase(models.Model):
+
+    @classmethod
+    def log_model_no_guid_bulk_update(cls, models, user, update_type):
+        if models:
```

Comment:
per my previous review, it is not readable to have the entire contents of a function nested inside an `if` block. You should explicitly return if there are no models and leave the rest of the function at top level

Reply:
Sounds good.